src/gui/Src/BasicView/HexDump.h

```diff
@@ -217,6 +218,7 @@ public slots:
     QAction* mCopyAddress;
     QAction* mCopyRva;
     QAction* mCopySelection;
+    QAction* mCopyFileOffset;

```

Comment:
This action should be created with `makeShortcutAction` in `CPUDump.cpp` not over here.